### DISCUSSION TITLE
How to remove some feature unused?

### DISCUSSION THREAD

**Discussion by anhnv-nhsv** [brocoders/nestjs-boilerplate#1453] [Q&A] [answered]
Hi guys,

I like this boilerplate because it's quite fully code and documents.
I want to use this boilerplate for my project, but my project is not complex, it's quite simple. 
All I need are:
- MySQL
- Basic MVC like SpringBoot
- Custom response data
- Interceptor, middleware, exception handlers
- Do not need Docker, Mailer

So, how can I remove unused features, plz?
Thanks!

**Comment by Shchepotin** [comment, marked as answer]
@anhnv-nhsv Currently you can do this manually, but we are working on a CLI tool for removing unused features 

UPD: `npm run app:config`

**Reply by anhnv-nhsv** [reply]
I look forward to receiving the latest version with more powerful CLI. Thanks team!
